schemas/iso19139/src/main/plugin/iso19139/layout/config-editor.xml

```diff
@@ -3192,10 +3190,39 @@
                    data-lang="lang"/>
 
         <directive data-gn-suggestion-list=""/>
+
         <directive data-gn-need-help="user-guide/describing-information/creating-metadata.html"/>
       </sidePanel>
 
       <tab id="default" default="true" mode="flat">
+
+        <!--
+        Example for file upload in the main form
+        <section>
+          <field xpath="/gmd:MD_Metadata/gmd:identificationInfo//gmd:title"/>
+
+          * Manage uploaded file with a WWW:DOWNLOAD protocol
+          <directive data-gn-file-uploader=""
+                     data-type="onlines"/>
+
+          * Same but with custom titles and icons
+          <directive data-gn-file-uploader=""
+                     data-title="Resource for download"
```

Comment:
Does this support translation keys?

Reply:
You could put text or a key that you can set in database translation.